Lesson_16/l16_h01_class.py

```diff
@@ -0,0 +1,77 @@
+"""
+HW:16[Volkov Anton].
+
+# write class for bank card.
+# Class should reflect card lifecycle, card operations etc.
+# use CVV, PIN, Name, Surname , end date, card_num as initial params
+# class should have in addition to common logic some class attributes,
+# as minimum one classmethod and
+# as minimum  one staticmethod, two or more getters/setters
+# do not forget about block if __name__ == '__main__':
+"""
+
+
+class MonoBankCard:
+    """Class for creating cards."""
+
+    _lvl_card = None
+
+    def __init__(self, card_num, cvv, pin, name, surname, end_date):
+        """Initialize class."""
+        self._card_num = card_num
+        self._cvv = cvv
```

Comment:
CVV and PIN fro sure should be NOT protected but private attributes. Just try to think close to real conditions and real life flow.